### PR TITLE
Clarify external kubelet server approver requirements

### PR DESCRIPTION
**Pull request by liggitt**
/sig auth

cc @chardch

### CODE REVIEW

**Comment by bradtopol** [comment]
Hi @chardch can you serve as the technical reviewer for the PR?  Just add a /lgtm if it is technically correct.  Thanks!
/assign

File: content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md [hunk 1]
```diff
@@ -415,6 +415,17 @@ approve node _serving_ certificates for [security
 reasons](https://github.com/kubernetes/community/pull/1982). To use
 `RotateKubeletServerCertificate` operators need to run a custom approving
 controller, or manually approve the serving certificate requests.
+
+A deployment-specific approver for kubelet serving certificates should typically only approve CSRs which:
```
Comment (kbhawkey) [inline content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md:419]: - Is "A deployment-specific approver" referring to a custom controller? What do you think about adding "controller".

Comment (liggitt) [reply content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md:419]: it's philosophically a controller, but could be a manual process (e.g. "run `kubectl describe csr/$name`, review, then run `kubectl certificate approve $name`")

Comment (kbhawkey) [reply content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md:419]: OK. "approver" sounds vague. Perhaps this is previously defined in the page.

Comment (liggitt) [reply content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md:419]: would changing it to "A deployment-specific approval process for kubelet serving certificates..." be better?

Comment (kbhawkey) [reply content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md:419]: Sure. Thanks!

Comment (liggitt) [reply content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md]: updated

**Comment by kbhawkey** [comment]
Reaching out to @chardch for review.

**Comment by kbhawkey** [comment]
👋 @liggitt . Can you recommend a technical reviewer?

Page preview:
https://deploy-preview-24418--kubernetes-io-master-staging.netlify.app/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation

/lgtm

**Comment by liggitt** [comment]
/cc @munnerz @deads2k 
for technical review

**Comment by kbhawkey** [comment]
Reviewers if there are more comments or corrections, feel free to open a new pull request.
Thanks @liggitt 
/approve

**Pull request merged**